etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/addSendingFacilityToMessageHeader.java

```diff
@@ -0,0 +1,24 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom;
+
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
+import gov.hhs.cdc.trustedintermediary.etor.metadata.EtorMetadataStep;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.FhirResource;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhirTransformation;
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiOrderConverter;
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetadata;
+import java.util.Map;
+import org.hl7.fhir.r4.model.Bundle;
+
+public class addSendingFacilityToMessageHeader implements CustomFhirTransformation {
+
+    private final MetricMetadata metadata =
+            ApplicationContext.getImplementation(MetricMetadata.class);
+
+    @Override
+    public void transform(FhirResource<?> resource, Map<String, String> args) {
+        Bundle bundle = (Bundle) resource.getUnderlyingResource();
+        HapiOrderConverter.addSendingFacilityToMessageHeader(
```

Comment:
@luis-pabon-tf  I was thinking we can add new transformations in a new PR as this one is big enough